components/NewsCard.vue

```diff
@@ -1,20 +1,38 @@
 <template>
-  <v-card class="mx-auto" max-width="500">
+  <v-card class="mx-auto" width="500">
     <v-img class="white--text align-end" height="200px" :src="news?.urlToImage">
-      <v-card-title>#{{ index }} {{ news?.title }}</v-card-title>
+      <v-card-title>
+        <a :href="news?.url" class="text-decoration-none">
+          #{{ index }} {{ news?.title }}</a
+        >
+      </v-card-title>
 
       <template #placeholder>
         <v-row class="fill-height ma-0" align="center" justify="center">
           <v-progress-circular
+            v-if="news?.urlToImage"
             indeterminate
             color="grey lighten-5"
           ></v-progress-circular>
+          <v-col v-else class="black fill-height">
+            <v-icon class="col-12" min-height="200px"> mdi-note-alert </v-icon>
```

Comment:
class="col-12" wont work here

Reply:
```css
.col-12{
    width: 100%;
    padding: 12px;
}
``` this is the definition of .col-12 so we can use it anywhere.

Reply:
I think the definition of is 
```scss
.col-12 {
    flex: 0 0 100%;
    flex-grow: 0;
    flex-shrink: 0;
    flex-basis: 100%;
    max-width: 100%;
    width: 100%;
    padding: 12px;
}```

So the uses depends.